examples/cc/utils/docker_runtime.py

```diff
@@ -0,0 +1,605 @@
+"""
```

Comment:
This file looks like a general util. Any third-party lib we can directly leverage? Such as pydocker?